pre_commit/languages/pcre.py

```diff
@@ -19,4 +19,11 @@ def run_hook(prefix, hook, file_args):
 
     # Grep usually returns 0 for matches, and nonzero for non-matches so we
     # negate it here.
-    return xargs(cmd, file_args, negate=True)
+    return xargs(
+        cmd,
+        file_args,
+        negate=True,
+        # TODO: does it actually make sense to parallelize this hook, given
+        # it's probably I/O bound?
```

Comment:
yeah, probably make this always synchronous is fine -- pcre is going away eventually anyway (for pygrep -- though I assume below I'm about to look at pygrep anyway!)